extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-logs.css

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-story-dev-tools-tab-logs {
+  overflow-y: scroll !important;
+  margin-top: 30px !important;
```

Comment:
nit:
I wonder if we want this minimum space between the header and content.
Maybe it's a part of `i-amphtml-story-dev-tools-header`.

Reply:
Now it's in the header padding and the element padding that it needs in this case 👍